.eslintrc.js

```diff
@@ -1,3 +1,12 @@
+// TODO: Enable this at some point.
```

Comment:
what prevents it from being enabled right now?

Reply:
There are some exceptions still.